config/vmdb.tmpl.yml

```diff
@@ -344,6 +344,8 @@ workers:
           :poll_method: :escalate
         :ems_metrics_collector_worker_kubernetes:
           :poll_method: :escalate
+          :hawkular_port: 5000
```

Comment:
is it safe to hardcode port here? perhaps it should be part of the configuration screen of the provider, similar to how it's done for rhev/ovirt provider (there's a metrics section in provider summary screen)